### PR TITLE
Improve messages and localization for Oracle account management Web UI

### PR DESCRIPTION
**Pull request by jsoref**
It's a terrible idea to reuse a string in multiple contexts. I've split the strings as identifiers to make it easier for people to evolve them independently. I've also adjusted the English translations a bit (albeit not independently).

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
retriggering CI

**Comment by oleg-nenashev** [comment]
Will merge it before the next release, even if https://github.com/jenkinsci/jdk-tool-plugin/pull/4#discussion_r304881691 is not addressed

File: src/main/resources/hudson/tools/JDKInstaller/DescriptorImpl/credentialOK.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+title=Oracle JDK Credentials saved
```
[the remark this answers is not in the document]
Comment (jsoref) [reply src/main/resources/hudson/tools/JDKInstaller/DescriptorImpl/credentialOK.properties]: I have no idea. I'd rather this be merged and let someone adjust it later.

[the remark this answers is not in the document]
Comment (oleg-nenashev) [reply src/main/resources/hudson/tools/JDKInstaller/DescriptorImpl/credentialOK.properties]: ok

**Pull request merged**